src/js/task.test.js

```diff
@@ -41,7 +42,51 @@ describe('Remove task', () => {
     task.addtask('Microverse1', false, 1);
     task.addtask('Microverse2', false, 2);
     task.removetask(1);
-
     expect(task.taskDtata).toHaveLength(2);
   });
+});
+
+describe('Editing', () => {
+  test('Editing task', () => {
+    const task = new Todolist();
+    task.addtask('Microverse', false, 0);
+    task.addtask('Microverse1', false, 1);
+    task.addtask('Microverse2', false, 2);
+    task.updatetask(2, 'Microverse2');
+    task.taskDtata[2].index = 6;
+    // task.removetask(1);
+    expect(task.taskDtata[2].index).toEqual(6);
+  });
+});
```

Comment:
As requested in the project requirements, please make sure to test the following functions:
![image](https://user-images.githubusercontent.com/42799579/207925203-6f9839f9-ad56-471b-9c6a-afd397ce39fa.png)

Nice job testing the last two functions 😄! Kindly test your editing description as well. Currently, this function is testing the `index` not the `description` property of your task 😉.